<launch>
	<!--仿真中的地图编号-->
	<param name="sim_map_id" type="int" value="1"/>
	<!--交汇路口安全距离/-->
	<param name="mtg_dist" type="double" value="2.0"/>
	<!--前方安全距离/-->
	<param name="flw_dist" type="double" value="1.6"/>
	<!--激光安全距离/-->
	<param name="laser_dist" type="double" value="2.0"/>
	<!--飞行速度/-->
	<param name="dsr_vel" type="double" value="2.0"/>

	<node name = "uav0" pkg="iusc_maze" type="iusc_maze_node" output="log">
		<param name="initial_x" type="double" value="-1.0"/>
		<param name="initial_y" type="double" value="22.5"/>
		<param name="uav_id" type="int" value="0"/>
	</node>
	<node name = "uav1" pkg="iusc_maze" type="iusc_maze_node" output="log">
		<param name="initial_x" type="double" value="-1.0"/>
		<param name="initial_y" type="double" value="15.0"/>
		<param name="uav_id" type="int" value="1"/>
	</node>
	<node name = "uav2" pkg="iusc_maze" type="iusc_maze_node" output="log">
		<param name="initial_x" type="double" value="-1.0"/>
		<param name="initial_y" type="double" value="7.5"/>
		<param name="uav_id" type="int" value="2"/>
	</node>
	<node name = "uav3" pkg="iusc_maze" type="iusc_maze_node" output="log">
		<param name="initial_x" type="double" value="-3.0"/>
		<param name="initial_y" type="double" value="22.5"/>
		<param name="uav_id" type="int" value="3"/>
	</node>
	<node name = "uav4" pkg="iusc_maze" type="iusc_maze_node" output="log">
		<param name="initial_x" type="double" value="-3.0"/>
		<param name="initial_y" type="double" value="15.0"/>
		<param name="uav_id" type="int" value="4"/>
	</node>
	<node name = "uav5" pkg="iusc_maze" type="iusc_maze_node" output="screen">
		<param name="initial_x" type="double" value="-3.0"/>
		<param name="initial_y" type="double" value="7.5"/>
		<param name="uav_id" type="int" value="5"/>
	</node>
	
	<node name = "rviz" pkg="rviz" type="rviz" args="-d $(find iusc_maze)/rviz.rviz" required="true"/>
</launch>
